plugins/sentry/lib/samson_sentry/samson_plugin.rb

```diff
@@ -24,5 +24,5 @@ class SamsonPlugin < Rails::Engine
 
 Samson::Hooks.callback :error do |exception, **options|
   sentry_options = options.slice(:contexts, :extra, :tags, :user, :level, :fingerprint)
-  Sentry.capture_exception(exception, sentry_options)
+  Sentry.capture_exception(exception, **sentry_options)
```

Comment:
The `Sentry.capture_exception` method accepts keyword arguments. Update this method call to match.

https://github.com/getsentry/sentry-ruby/blob/1d8054837bedda96342c5f7b4bcfe9d87614c4da/sentry-ruby/lib/sentry-ruby.rb#L398